sql/src/main/java/io/crate/protocols/postgres/PostgresNetty.java

```diff
@@ -75,11 +105,110 @@ public ChannelPipeline getPipeline() throws Exception {
                 return pipeline;
             }
         });
-        bootstrap.bind(new InetSocketAddress("127.0.0.1", 4242));
+
+        boundAddress = createBoundAddress();
+    }
+
+    private BoundTransportAddress createBoundAddress() {
+        // Bind and start to accept incoming connections.
+        InetAddress hostAddresses[];
+        try {
+            hostAddresses = networkService.resolveBindHostAddresses(bindHosts);
+        } catch (IOException e) {
+            throw new BindPostgresException("Failed to resolve host [" + Arrays.toString(bindHosts) + "]", e);
```

Comment:
mostly `SocketException`, `UnknownHostException` and other IO errors that may occur when resolving the address